packages/presentation/lib/l10n/app_en.arb

```diff
@@ -0,0 +1,12 @@
+
+{
+  "appTitle": "App Title",
+  "textWithParam": "Text with {param}",
+  "@textWithParam": {
+    "placeholders": {
+      "param": {
+        "type": "String"
+      }
+    }
+  }
```

Comment:
why do u need it?

Reply:
I use them just for test when set up localization